README.md

```diff
@@ -8,19 +8,12 @@ A moderation bot for discord communities.
 
 You will need the following to run Safety Jim
 
-- [ ] JDK 8
-- [ ] Gradle
-- [ ] PostgreSQL
+- Docker
 
 ## Development
-To develop Jim, you need to first make sure you have a local instance of postgresql running with schemas properly setup. Gradle will connect to your postgresql instance
-by using parameters defined (here)[build.gradle#L40] so feel free to change these to fit your own setup. You can find schema generation script (here)[schema.sql].
+Make a copy of `config_example.toml` called `example.toml` and fill in anything you need. To modify the username, password, and database name, set the environment variables `POSTGRES_USER`, `POSTGRES_PASSWORD`, and `POSTGRES_DB` respectively, and update them accordingly in the `config.toml` file, but if you just want to test the bot, leaving the defaults is fine.
```

Comment:
As Jim uses `config.toml` file we can explicitly tell the user to make the copy's file name that.